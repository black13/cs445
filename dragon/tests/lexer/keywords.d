div
mod
and
or
not
function
procedure
begin
end
integer
real
array
of
program
do
else
if
then
var
while boop
boolean
string
char
