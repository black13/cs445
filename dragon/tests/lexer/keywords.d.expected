DIV
MOD
AND
OR
NOT
FUNCTION
PROCEDURE
TBEGIN
END
INTEGER
REAL
ARRAY
OF
PROGRAM
DO
ELSE
IF
THEN
VAR
WHILE
ID(boop)
BOOLEAN
STRING
CHAR
EOF
-- done dumping tokens --
